src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -893,7 +893,6 @@
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Advapi32\Interop.RegFlushKey.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Advapi32\Interop.RegistryConstants.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Advapi32\Interop.RegOpenKeyEx.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Advapi32\Interop.RegQueryInfoKey.cs" />
```

Comment:
Who uses this file anymore?

Reply:
corefx does.  I'm going to separately move it out of the shared folder in corefx.